drv.c

```diff
@@ -230,6 +235,9 @@ struct combination *drv_get_combination(struct driver *drv, uint32_t format, uin
 	if (format == DRM_FORMAT_NONE || use_flags == BO_USE_NONE)
 		return 0;
 
+	if (use_flags & BO_USE_SCANOUT)
+		use_flags &= ~BO_USE_SCANOUT;
+
```

Comment:
such change is ambiguous, it makes BO_USE_SCANOUT useless. It is better refine the change.